interfaces/functions.yaml

```diff
@@ -550,6 +550,12 @@ igraph_distances_floyd_warshall:
         GRAPH graph, OUT MATRIX res, EDGEWEIGHTS weights=NULL, NEIMODE mode=OUT
     DEPS: weights ON graph
 
+igraph_voronoi:
+    PARAMS: |-
+        GRAPH graph, OUT VECTOR_INT membership, OUT VECTOR distances,
+        VECTOR_INT generators, VECTOR weights, NEIMODE mode=OUT, VORONOI_TIEBREAKER tiebreaker=RANDOM
```

Comment:
* `generators` should be of type `VERTEX_INDICES`.
* `weights` should be of type `EDGEWEIGHTS` with a default of `NULL`